aion/web/elm/src/Auth/View.elm

```diff
@@ -17,17 +18,47 @@ import Toasty.Defaults
 
 authView : Model -> Html Msg
 authView model =
-    div [ class "auth-container" ]
-        [ case model.authData.unauthenticatedView of
-            LoginView ->
-                loginFormView model.authData.loginForm
-
-            RegisterView ->
-                registrationFormView model.authData.registrationForm
-        , br [] []
-        , authFormToggle model.authData.formMsg
-        , Toasty.view toastsConfig Toasty.Defaults.view ToastyMsg model.toasties
-        ]
+    let
+        unauthenticatedView =
+            model.authData.unauthenticatedView
+
+        loginForm =
+            model.authData.loginForm
+
+        registrationForm =
+            model.authData.registrationForm
+
+        formMsg =
+            model.authData.formMsg
+    in
+        Grid.container [ class "auth-container" ]
+            [ Grid.container []
+                [ Grid.row []
+                    [ Grid.col []
+                        [ case unauthenticatedView of
+                            LoginView ->
+                                loginFormView loginForm formMsg
+
+                            RegisterView ->
+                                registrationFormView registrationForm formMsg
+                        ]
+                    , Grid.col []
+                        [ h2 []
+                            [ text "Welcome to "
+                            , span [ class "highlight-aion" ] [ text "Aion" ]
+                            , text "!"
+                            ]
+                        , div
+                            [ class "auth-page-content" ]
+                            [ p [] [ text "Aion is an e-learning platform written in Elixir and Elm based on real-time gameplay." ]
```

Comment:
These hardcoded strings could also be done as a separate function, you could think about putting them to `Constants` (or wherever we keep strings). Once again, when you look at this line, you don't care about each specific word, `platformDescriptionStr` (or any other more fitting name, just throwing an idea) would be just enough, you only need to get the idea of what kind of message will be there. 

^ That's why I would move it out to make this function more readable and separate the strings themselves and the way they are displayed.